crates/dids/methods/web/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "did-web"
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
Can you revert this change? I think we bump the version number on release. @sbihel?